### PR TITLE
ASM-7793 Create symlink to most recent CompCU.jar

### PR DESCRIPTION
**Pull request by gavin-scott**
The dell-compellent module includes several versions of the CompCU jar
file. Previously code changes had to be made whenever a new version of
the jar was added. This change automatically creates a symlink in the
latest dell-compellent rpm pointing to the newest version of the
CompCU jar file.

Separate code changes will be made to the dell-compellent module and
asm-core java code to always use the symlinked jar file to ensure they
always get the newest version of the jar.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -50,16 +50,52 @@ task rpm() << {
...
+
 def buildModuleRpm(moduleName,moduleVersion,puppetModulesFolder='/etc/puppetlabs/puppet/modules',rpmDestinationDirectory=(new File("${buildDir}/distributions"))) {
 
     if (! rpmDestinationDirectory.isDirectory()) {
         rpmDestinationDirectory.mkdirs()
     }
 
-    org.freecompany.redline.Builder rpmBuilder = new org.freecompany.redline.Builder()
-
+    org.redline_rpm.Builder rpmBuilder = new org.redline_rpm.Builder()
```
Comment (gavin-scott) [inline build.gradle]: The package change is due to the upgraded redline library.

File: build.gradle [hunk 2]
```diff
@@ -68,34 +104,46 @@ def buildModuleRpm(moduleName,moduleVersion,puppetModulesFolder='/etc/puppetlabs
...
            }
        }
 
-    rpmFileTree.visit { element->
+    def compCuInfo = null
+
+    rpmFileTree.visit { element ->
         def visitedFile = element.file
-		logger.debug( "File name and Path = ${element.path}  ${element.file.name}")
+        logger.debug("File name and Path = ${element.path}  ${element.file.name}")
```
Comment (gavin-scott) [inline build.gradle]: Sorry about the whitespace changes, but seemed goot o get rid of the tabs.

File: build.gradle [hunk 3]
```diff
@@ -110,15 +158,5 @@ def buildModuleRpm(moduleName,moduleVersion,puppetModulesFolder='/etc/puppetlabs
...
-   // If we are building the dell-bmc module, we need to install the ipmiflash rpm
-    if (moduleName == 'dell-bmc') {
-      rpmBuilder.addDependencyMore('dell-pec-ipmiflash','2013.02.11')
-    }
-    if (moduleName == 'dell-compellent') {
-      rpmBuilder.addDirectory("/opt/Dell/ASM/logs/compellent", 0775, Directive.NONE, project.puppet_user, project.puppet_user)
-    }
-    if (moduleName == 'dell-equallogic') {
-      rpmBuilder.addDirectory("/var/lib/net-snmp/mib_indexes", 0775, Directive.NONE, project.puppet_user, project.puppet_user)
-    }
```
Comment (gavin-scott) [inline build.gradle]: Oops this block shouldn't be here, will fix.

**Comment by gavin-scott** [comment]
This should be good to go @hribab

**Comment by hribab** [comment]
👍

**Comment by gavin-scott** [comment]
Merging based on review with @hribab

**Pull request merged**